public/js/main.js

```diff
@@ -1,58 +1,42 @@
-/* global  XMLHttpRequest */
-
-(function () {
-  var url = 'https://nazareth-open-tourism-platform.herokuapp.com/events';
-  function makeRequest (url, callback) {
-    var xhr = new XMLHttpRequest();
-    xhr.onreadystatechange = function () {
-      if (xhr.readyState === 4 && xhr.status === 200) {
-        callback(null, JSON.parse(xhr.responseText));
-      } else if (xhr.status === 500) {
-        var errorMessage = new Error('Server Error! Status: ' + xhr.status);
-        callback(errorMessage, null);
-      }
-    };
-    xhr.open('GET', url, true);
-    xhr.send();
-  };
-
-  function renderEvents (error, eventsArray) {
-    var listPage = document.getElementById('list-page-content');
-    var eventsSection = document.getElementById('events-section');
-
-    if (error) {
-      eventsSection.innerHTML = error;
-    }
-
-    var filterEvents = eventsArray.filter(function (event) {
-      if (event.en) {
-        return true;
-      }
-      return false;
-    });
-
-    filterEvents.forEach(function (event) {
-      var eventContainerDiv = document.createElement('div');
-      eventContainerDiv.className = 'container-div';
-      var h3Element = document.createElement('h3');
-      h3Element.className = 'event text-primary';
-      var h4Element = document.createElement('h4');
-      h4Element.className = 'placeTime text-muted';
-      var aElement = document.createElement('a');
-      var id = event._id;
-      aElement.href = '/events/' + id;
-      aElement.className = 'event-link';
-
-      h3Element.innerHTML = event.en.name;
-      h4Element.innerHTML = event.placeId.en.name + '<br>' + new Date(new Date(event.startTime).getTime()).toLocaleTimeString() + ' - ' + new Date(new Date(event.endTime).getTime()).toLocaleTimeString();
-
-      aElement.appendChild(h3Element);
-      aElement.appendChild(h4Element);
-      eventContainerDiv.appendChild(aElement);
-      eventsSection.appendChild(eventContainerDiv);
-    });
-    listPage.appendChild(eventsSection);
-  }
-
-  makeRequest(url, renderEvents);
-})();
+/* global renderModule Cal */
+function dateClickHandler (e) {
+  var plusDay = parseInt(e.target.dataset.date.split('-')[2]) + 1;
+  var plusDate = e.target.dataset.date.split('-');
```

Comment:
I would recommend making a function which adds a certain number of days to a date.
e.g.
```js
addDays('2015-12-30', 2) // should return '2016-01-01'
```

Reply:
it may be useful to know that you can do `new Date` in javascript with 'wrong' numbers
e.g. try this in the console
```js
new Date(2017, 1, 40)
```

Reply:
...or [momentjs](http://momentjs.com/)

Reply:
And I still think you should show events for a full week after the date you select... @shiryz how can we decide on this?

Reply:
@mattlub , we should open an issue and assign Silke to decide which she prefers

Reply:
@mattlub @shiryz who is we in this case ?? :maple_leaf: 
is anyone going to open it or should I ?